alfalfa_web/Dockerfile

```diff
@@ -17,5 +17,5 @@ COPY lib/dbops.js /root/server/
 
 ARG NODE_ENV
 RUN npm run build
-
+RUN ls -alh build/app/
```

Comment:
this looks harmless, but im just curious what is the purpose?

Reply:
Debugging that app.bundle.js weird filesize stuff.  I just decided to leave it in.